packages/@react-stately/color/package.json

```diff
@@ -19,7 +19,6 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.6.2",
-    "@react-aria/i18n": "3.2.0",
     "@react-aria/utils": "^3.2.1",
```

Comment:
unsure what to do about this one, Color uses `clamp` from here
I feel like clamp should be in stately utils though?